pkg/webui/components/breadcrumbs/breadcrumb/breadcrumb.styl

```diff
@@ -31,8 +31,11 @@
 
 .last
   one-liner()
+  display: flex
+  align-items: center
   color: $tc-deep-gray
   min-height: 18px
+  font-weight: $fw.bold
```

Comment:
This is not sufficient from what I can see. The vertical position is still inconsistent which is due to the different heights of the breadcrumb parts. To fix this, I believe it's best to turn all breadcrumb-parts into `display: flex` and fixing their heights to `1rem`.